### PR TITLE
Add initial setup code for table.

### PR DESCRIPTION
**Pull request by JoshDunnDev**

### CODE REVIEW

**Review by mattcollier — APPROVED** [review]

File: BrQTable.vue [hunk 1]
```diff
@@ -1,14 +1,132 @@
...
 export default {
   name: 'BrQTable',
-  props: {}
+  props: {
+    loadingData: {
+      type: Boolean,
+      default: false,
+      required: false
+    },
+    tableColumns: {
```
Comment (gannan08) [inline BrQTable.vue]:
```suggestion
    columns: {
```

File: BrQTable.vue [hunk 2]
```diff
@@ -1,14 +1,132 @@
...
 /*!
  * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
  */
-'use strict';
-
 export default {
   name: 'BrQTable',
-  props: {}
+  props: {
+    loadingData: {
```
Comment (gannan08) [inline BrQTable.vue]:
```suggestion
    loading: {
```

File: BrQTable.vue [hunk 3]
```diff
@@ -1,14 +1,132 @@
...
+      type: Boolean,
+      default: false,
+      required: false
+    },
+    tableColumns: {
+      type: Array,
+      required: true
+    },
+    tableData: {
```
Comment (gannan08) [inline BrQTable.vue]:
```suggestion
    data: {
```

**Review by gannan08 — CHANGES_REQUESTED** [review]
The name of QTable props should be preserved. Ideally, we should be extending the component such that we have access to the same QTable API; props and all.

**Review by gannan08 — APPROVED** [review]
LGTM. The renaming of the props will cause any dependent library to break. We will need PRs for anything using br-q-table.

**Comment by JoshDunnDev** [comment]
> LGTM. The renaming of the props will cause any dependent library to break. We will need PRs for anything using br-q-table.

Well this needs to be merged into master so we can properly make PRs. I will add a fix to the wallet now.

**Pull request merged**